firefox_services_tests/helpers/utils.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import json
+import time
+import urllib2
+
+
```

Comment:
SInce these are all `staticmethod`s , you don't need to have them in a class. Python treats each file as a module so you can do remove `ServiceUtils.*` type calls below and just call the method directly. 

In the code that uses the class you just call it directly off the module

so
```python
from firefox_services_tests.helpers.utils import ServiceUtils
ServiceUtils.some_method
``` 
changes to 
```python
from firefox_services_tests.helpers import utils
utils.some_method
```